app/src/main/java/ca/ualbert/cs/tasko/ViewTaskDetailsActivity.java

```diff
@@ -153,6 +154,23 @@ public void onClick(DialogInterface dialog, int id) {
         });
     }
 
+    @Override
+    protected void onResume() {
+        super.onResume();
+        try {
+            String taskID = getIntent().getExtras().getString("TaskID");
+            currentTask = dm.getTask(taskID, this);
+        } catch (NullPointerException e) {
+            Log.i("Error", "TaskID not properly passed");
+        } catch (NoInternetException e) {
+            e.printStackTrace();
+        }
+
+        if (currentTask.getStatus() == TaskStatus.ASSIGNED) {

```

Comment:
If I return to this activity after editing I get a null pointer Error on line 169